sdk/appcenter/src/main/java/com/microsoft/appcenter/channel/DefaultChannel.java

```diff
@@ -134,9 +141,10 @@ public DefaultChannel(@NonNull Context context, String appSecret, @NonNull LogSe
      * @param persistence      Persistence object for dependency injection.
      * @param ingestion        Ingestion object for dependency injection.
      * @param appCenterHandler App Center looper thread handler.
+     * @param authTokenContext Authorization token context.
      */
     @VisibleForTesting
-    DefaultChannel(@NonNull Context context, String appSecret, @NonNull Persistence persistence, @NonNull Ingestion ingestion, @NonNull Handler appCenterHandler) {
+    DefaultChannel(@NonNull Context context, String appSecret, @NonNull Persistence persistence, @NonNull Ingestion ingestion, @NonNull Handler appCenterHandler, AuthTokenContext authTokenContext) {
```

Comment:
no need to be a dependency, context is already a dependency so you can init from context in constructor

our mocking framework is powerful enough in java to avoid boilerplate if that was the reason